pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -864,7 +864,7 @@ private void addWebServerHandlers(WebService webService,
         // Add metrics servlet
         webService.addServlet("/metrics",
                 new ServletHolder(metricsServlet),
-                false, attributeMap);
+                config.isAuthenticateMetricsEndpoint(), attributeMap);
```

Comment:
It looks like this change is not covered by tests

Reply:
I will add.